### PR TITLE
configure: Check for version of pytest

### PR DESCRIPTION
**Pull request by jsarenik**
Addresses #486

### CODE REVIEW

File: configure [hunk 1]
```diff
@@ -73,6 +73,7 @@ find_pytest()
 }
 
 PYTEST=${PYTEST:-`find_pytest`}
+$PYTEST --version 2>&1 | grep -q "version 3" || unset PYTEST
```
Comment (rustyrussell) [inline configure]: We should NOT override if they set PYTEST manually, though.  I think this logic needs to be in find_pytest?

**Comment by jsarenik** [comment]
I have tested with `PYTEST=/bin/true ./configure`

**Comment by practicalswift** [comment]
utACK 59c54341fbf198d3475916eeb03054a1cb5bafbf

**Comment by jsarenik** [comment]
@practicalswift what does `ut` in `utACK` mean?

**Comment by cdecker** [comment]
ACK 59c5434

**Pull request merged**